### ISSUE TITLE
can't jump if file path has chinese

### ISSUE THREAD

**Issue by tiantao433** [austin-----/code-gnu-global#3]
can't jump to definition if file path has chinese. my os is win10

**Comment by austin-----** [comment]
@tiantao433, what is the output if you run the following command in the directory where your GTAGS file is (replace FUNCTION with the one you are jumping to):

```
global --encode-path " " -xa FUNCTION
```

**Comment by tiantao433** [comment]
the command output is right, but jump failed in vscode
![baidushurufa_2015-12-18_23-33-36](https://cloud.githubusercontent.com/assets/6522598/11900433/d62c0a1c-a5e0-11e5-8af5-7553dcaa35ea.jpg)
![baidushurufa_2015-12-18_23-31-38](https://cloud.githubusercontent.com/assets/6522598/11900423/cbdc2b82-a5e0-11e5-971d-87e6399b2841.jpg)

**Comment by austin-----** [comment]
Looks like nodejs process always expect UTF8 from output. For code page other than English on Windows, we may need to manually transcode it into UTF8. Will look into it.

**Comment by austin-----** [comment]
Should have been fixed in 0.1.6. Please look into the README on how to set the encodings in the User Settings. Hint:

```
{
    'codegnuglobal.encoding': 'cp936'
}
```
